packages/acquirer-backend/src/routes/userControllers/forgotPassword.ts

```diff
@@ -0,0 +1,123 @@
+import { type Response } from 'express'
+import jwt from 'jsonwebtoken'
+import ms from 'ms'
+import logger from '../../services/logger'
+import { audit } from '../../utils/audit'
+import { AppDataSource } from '../../database/dataSource'
+import { AuditActionType, AuditTrasactionStatus } from 'shared-lib'
+import { type AuthRequest } from '../../types/express'
+import { PortalUserEntity } from '../../entity/PortalUserEntity'
+import { readEnv } from '../../setup/readEnv'
+import { sendForgotPasswordEmail } from '../../utils/sendGrid'
+import { JwtTokenEntity } from '../../entity/JwtTokenEntity'
+import { EmailVerificationTokenEntity } from '../../entity/EmailVerificationToken'
+
+const JWT_SECRET = readEnv('JWT_SECRET', 'secret') as string
+
+/**
```

Comment:
Is this commented section still required ?

Reply:
yes it is still required... openapi documentation will scan through `routes` directory and generate the open api file...
if i remove the comments, controller doc will also be removed automatically.

This is where generation happens.
https://github.com/mojaloop/merchant-registry-svc/blob/main/packages/acquirer-backend/src/setup/openapiSpecConfig.ts

Reply:
If it is required, why is it commented?

Reply:
hi @elnyry-sam-k  they are special annotations that are used by the OpenAPI documentation generator to create an accurate and up-to-date API specification document, It is needed.